source/mainnet/net/guides/tokens.rst

```diff
@@ -0,0 +1,80 @@
+.. include:: ../../variables.rst
+.. _tokens:
+
+========================
+Tokens in the wallet
+========================
+
+You can add, inspect, and send tokens in the |bw|.
+
+.. dropdown:: |bw|
+
+    To access tokens and token transactions, click Tokens |token-bw|. Tabs appear for tokens.
+
+    .. dropdown:: Add tokens
+
+        #. Click **Manage**.
+
+            .. image:: ../images/browser-wallet/manage-tokens.png
+                :width: 50%
+
+        #. Enter a smart contract address on which to look for tokens. Click **Choose contract**. If you get an error that says **Invalid contract address** this means that the smart contract doesn’t exist, the smart contract doesn’t live up to the CIS-2 standard, or the smart contract doesn’t contain any tokens.
```

Comment:
"smart contract address" -> "smart contract index"

Also we don't give an error saying "Invalid contract address", but rather different errors depending on the issue:

- 'No contract found on index',
- 'Contract does not support CIS-0',
- 'Contract does not support CIS-2',
- 'No tokens found in contract',

That might be too much to fit in there though.

```suggestion
        #. Enter a smart contract index on which to look for tokens. Click **Choose contract**. If you get an error, this means that the smart contract doesn’t exist, the smart contract doesn’t live up to the CIS-2 standard, or the smart contract doesn’t contain any tokens.
```